app/scripts/migrations/105.ts

```diff
@@ -112,6 +115,7 @@ function createInternalAccountsForAccountsController(
       },
       methods: ETH_EOA_METHODS,
       type: EthAccountType.Eoa,
+      scopes: [EthScope.Eoa],
```

Comment:
Rather than omitting the Scopes property, we are defaulting to the Eth EOA scope (similar to the Eth EOA type).